### PR TITLE
mocked config values so that it works with local foremast configs

### PR DESCRIPTION
**Pull request by ndcampbell**
I was having to rename my local config for tests to work with default values. I mocked these defaults and now tests still work with out changing local foremast configs.

### CODE REVIEW

File: tests/dns/test_dns.py [hunk 1]
```diff
@@ -31,6 +32,7 @@ def test_dns_creation(mock_get_details, mock_properties, mock_find_elb, mock_dns
     }
 
     # mock results
+    #mock_domain.value = 'example.com'
```
Comment (e4r7hbug) [inline tests/dns/test_dns.py]: Remove comment.

File: tests/iam/test_iam_construct.py [hunk 2]
```diff
@@ -68,9 +72,10 @@ def test_construct_cloudwatchlogs(requests_get):
 
 @mock.patch('foremast.utils.credentials.API_URL', 'http://test.com')
 @mock.patch('foremast.utils.credentials.requests.get')
-def test_construct_s3(requests_get):
+@mock.patch('foremast.utils.templates.TEMPLATES_PATH', None)
```
Comment (e4r7hbug) [inline tests/iam/test_iam_construct.py]: Remove `patch` since `fixture` is working.

File: tests/iam/test_iam_construct.py [hunk 3]
```diff
@@ -95,9 +100,12 @@ def test_construct_s3(requests_get):
 
 @mock.patch('foremast.utils.credentials.API_URL', 'http://test.com')
 @mock.patch('foremast.utils.credentials.requests.get')
-def test_construct_s3_buckets(requests_get):
+@mock.patch('foremast.utils.templates.TEMPLATES_PATH', None)
```
Comment (e4r7hbug) [inline tests/iam/test_iam_construct.py]: Remove `patch` since `fixture` is working.

File: tests/iam/test_iam_construct.py [hunk 4]
```diff
@@ -95,9 +100,12 @@ def test_construct_s3(requests_get):
...
+def test_construct_s3_buckets(requests_get, get_base_settings):
     """Check S3 Policy with multiple Buckets listed."""
-    pipeline_settings = copy.deepcopy(BASE_SETTINGS)
+    #print(get_base_settings)
+    #pipeline_settings = copy.deepcopy(BASE_SETTINGS)
```
Comment (e4r7hbug) [inline tests/iam/test_iam_construct.py]: Remove comments.

File: tests/iam/test_iam_construct.py (hunk 2 above)
Comment (ndcampbell) [inline tests/iam/test_iam_construct.py]: These are necessary. The fixure and the test needs the patch

**Pull request merged**